pkg/steps/template.go

```diff
@@ -81,7 +82,9 @@ func (s *templateExecutionStep) Run(ctx context.Context, dry bool) error {
 		}
 	}
 
-	addArtifactsToTemplate(s.template)
+	if len(s.artifactDir) > 0 {
```

Comment:
Should this have failed a test somewhere?

Reply:
I'll add a test, but it only occurs when you don't ask for artifacts.